src/MailerMessage.php

```diff
@@ -36,6 +36,8 @@ class MailerMessage extends BaseMessage
 
     private $_customCampaign;
 
+    private $_customID;
```

Comment:
please use customId instead of customID, we always use lowerCamelCase

Reply:
Ok, Changed it.